python/sglang/srt/server_args.py

```diff
@@ -2419,6 +2420,11 @@ def add_cli_args(parser: argparse.ArgumentParser):
             action="store_true",
             help="Enable using mscclpp for small messages for all-reduce kernel and fall back to NCCL.",
         )
+        parser.add_argument(
```

Comment:
Please add hardware limit and world_size limit.

Reply:
Added.